src/settings/CustomFieldsSettings.js

```diff
@@ -26,17 +33,33 @@ const CustomFieldsSettings = ({
     canDelete: stripes.hasPerm('ui-users.settings.customfields.all'),
   };
 
+  const backendModuleName = 'users';
+  const entityType = 'user';
+
+
   if (!permissions.canView) {
     history.replace('/settings/users');
   }
 
   return (
-    <ViewCustomFieldsSettings
-      backendModuleName="users"
-      entityType="user"
-      redirectToEdit={redirectToEdit}
-      permissions={permissions}
-    />
+    <Switch>
+      <Route exact path={base}>
+        <ViewCustomFieldsSettings
+          backendModuleName={backendModuleName}
+          entityType={entityType}
+          redirectToEdit={redirectToEdit}
+          permissions={permissions}
+        />
+      </Route>
+      <Route exact path={`${base}/edit`}>
+        <EditCustomFieldsSettings
+          backendModuleName={backendModuleName}
+          entityType={entityType}
+          redirectToView={redirectToView}
+          permissions={permissions}
+        />
+      </Route>
```

Comment:
Is there no way to get these moved up to `src/settings/sections.js` where all other settings routes are defined?

Reply:
In this case there would be two links for Custom Fields in the 2nd pane, like so:
![image](https://user-images.githubusercontent.com/19309423/79867463-0a32b200-83e7-11ea-974e-b3cf60a9b6cf.png)
I was thinking of adding these routes to `sections`, but also defining a new `hidden` property for them so the edit route is not shown in the list.

Reply:
Hmm; I see why we don't want that. I guess we handle this elsewhere by appending a different query-string to a route, instead of using separate route paths, is that right? If so, is that possible here?

Reply:
Some apps do use a query string for separate routes, but in combination with the `<Layer>` component. So if we're not using `<Layer>` there's not much difference between route paths and query parameters